src/OpenConext/Profile/Api/AuthenticatedUserProviderInterface.php

```diff
@@ -20,7 +20,7 @@
 
 use OpenConext\Profile\Entity\AuthenticatedUser;
 
-interface AuthenticatedUserProvider
+interface AuthenticatedUserProviderInterface
```

Comment:
This PR contains a big amount of classes(interfaces) that simply had their name changed. This makes it a bit harder to find exactly what is changed.

Is this because a coding style updated?

Reply:
This is done to allow the Symfony autowire feature to work a little better. Having classes/interfaces with the same name in different namespaces make it harder to get it to work.